src/tests/unit/test_common/test_utils/test_data_structure/test_has_key_value_pairs.py

```diff
@@ -14,3 +14,8 @@ def test_target_contains_key_value_pair_returns_true(self):
     def test_target_not_contains_key_value_pair_returns_false(self):
         assert has_key_value_pairs(self.target, {"a": "2"}) is False
         assert has_key_value_pairs(self.target, {"b": "1"}) is False
+
+    def test_target_not_contains_key_returns_false(self):
+        assert has_key_value_pairs(self.target, {"x": "y"}) is False
+        assert has_key_value_pairs(self.target, {1: {"x": "y"}}) is False
+        assert has_key_value_pairs(self.target, {1: {"a": "1"}}) is False
```

Comment:
Could you try adding a test where `content=={"a": "1", "b": "1000"}`? I think the function might return true then, when it really should return false

Reply:
Added test with the content you specified. Pushed to ref now. 
Unit tests still run green!

Reply:
I see! I misunderstood. Thanks for clarifying. Will fix it now